src/docs.js

```diff
@@ -59,11 +63,49 @@ async function documenter(options) {
   // the stream was added
   // no more entries
   tarball.finalize();
-  // tarball.pipe(process.stdout);
 
   let gzip = zlib.createGzip();
   let tgz = tarball.pipe(gzip);
 
+  if (options.publish) {
+    let auth = new client.Auth({
+      credentials: options.credentials,
+    });
+
+    let creds = await auth.awsS3Credentials('read-write', 'taskcluster-raw-docs', 'testing/');
```

Comment:
This won't always be `testing/`. It needs to change based on the name of the project.